### ISSUE TITLE
Issue with mime field in canvas?

### ISSUE THREAD

**Issue by RussellMcOrmond** [crkn-rcdr/Access-Platform#32]
I'm not currently seeing the problem, so help needed:

```
tdr@4993cf6c5645:~/Databases$ curl http://iris.tor.c7a.ca:5984/canvas/69429%2Fc0kw57d2qf6n >~/a.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   237  100   237    0     0   1185      0 --:--:-- --:--:-- --:--:--  1185
tdr@4993cf6c5645:~/Databases$ node ./test.js -s schemas/canvas.json -f ~/a.json
[ { keyword: 'pattern',
    dataPath: '.master.mime',
    schemaPath: '#/properties/master/properties/mime/pattern',
    params: { pattern: '^w+/w+$' },
    message: 'should match pattern "^w+/w+$"' } ]
tdr@4993cf6c5645:~/Databases$ cat ~/a.json
{"_id":"69429/c0kw57d2qf6n","_rev":"1-2275da98952beeaf33e6a3b2809ea05f","master":{"size":140912,"path":"oocihm.8_06733_654/data/sip/data/files/oocihm.8_06733_654.0002.tif","mime":"image/tiff","height":3828,"width":2473},"source":"cihm"}
tdr@4993cf6c5645:~/Databases$ 

```

**Comment by RussellMcOrmond** [comment]
Seems the pattern needs to be updated.

```
    "mime": {
      "description": "A MIME type",
      "type": "string",
      "pattern": "^\\w+/\\w+$"
    },
```

**Comment by SaschaAdler** [comment]
Fixed in #33.
